docs/faq.rst

```diff
@@ -497,3 +497,4 @@ My bot's commands are not showing up!
    ``https://discord.com/oauth2/authorize?client_id=<client id>&scope=applications.commands+bot``.
    Alternatively, if you use :func:`utils.oauth_url`, you can call the function as such:
    ``oauth_url(<other options>, scopes=("bot", "applications.commands"))``.
+3. You may need to change server to update the slash commands on the client side
```

Comment:
This sentence makes no sense to me.

Reply:
sometimes, you have to go to another server and then come back to "sync" the commands on the discord application

Reply:
Documenting client bugs is outside of the scope of the documentation.

Reply:
Yeah I know but I think it's a good idea to write it because it took me a long time to realize that the problem was not my code, but the client.